.github/wiki/How-To-Contribute.md

```diff
@@ -0,0 +1,85 @@
+
+If you are eager to start contributing code right away, we have a list of [good first issues][gfi] that contain bugs which have a relatively limited scope. As you gain more experience and demonstrate a commitment to evolving React Native, you may be granted issue management permissions in the main repository.
+
+[gfi]: https://github.com/facebook/react-native/labels/good%20first%20issue
+
+# How to Contribute Code
+
+Code-level contributions to React Native generally come in the form of [pull requests](https://help.github.com/en/articles/about-pull-requests). The process of proposing a change to React Native can be summarized as follows:
+
+1. Fork the React Native repository and create your branch from `master`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. If you haven't already, complete the Contributor License Agreement ("CLA").
+7. Push the changes to your fork.
+8. Create a pull request to the core React Native repository.
+9. Review and address comments on your pull request.
+
+If all goes well, your pull request will be merged. If it is not merged, maintainers will do their best to explain the reason why.
+
+If this is your first time sending a pull request, we have created a [step-by-step guide to help you get started][first-pr].
+
+[first-pr]: https://github.com/facebook/react-native/wiki/Your-First-Pull-Request
+
+
+## Pull Requests
+
+Pull requests should be reviewed and approved using GitHub's review feature before they are considered eligible to be merged. While anyone has the ability to review and approve a pull request, we typically only consider a pull request ready to be merged when the approval comes from one of the [contributors](https://github.com/facebook/react-native/blob/master/ECOSYSTEM.md).
+
+Reviewing a pull request can take a considerable amount of time. In some cases, the review might require more time to perform than it took someone to write and submit their changes! It's therefore necessary to do some preliminary work to ensure each pull request is in a good state to be reviewed.
+
+
+### Handling Pull Requests
+
+A pull request should consist of three main sections:
+
+* A summary. This helps us understand the motivation behind the changes.
+* A changelog. This helps us write the release notes. It also serves as a brief summary of your changes.
+* A test plan. This might be the most important part of your pull request. It lets us know that you have verified the change properly, and helps us understand if you've considered any edge cases.
+
+Any one pull request may require a deeper understanding of some area of React Native that you may not be familiar with. Even if you don't feel like you are the right person to review a pull request, you may still help by adding labels or asking the author for more information.
```

Comment:
This reads a bit weirdly `Any one pull request` (maybe it's just too late, trying to proofread 😄 )

Reply:
I think removing "one" changes the meaning of the sentence, but I'm open to suggestions on how to reword this.

These are now in the wiki, which should make it easier for contributors to edit in place.